### PR TITLE
SAN-115: Refactor penalty-payment-api with repo and pipeline renaming…

### PR DESCRIPTION
**Pull request by dnair-ch**
… updates

### CODE REVIEW

File: dao/mongo.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/companieshouse/chs.go/log"
-	"github.com/companieshouse/lfp-pay-api-core/models"
+	"github.com/companieshouse/penalty-payment-api-core/models"
```
Comment (acallaghan-ch) [inline dao/mongo.go:10]: The 'lfp-pay-api-core' repo hasn't been renamed yet.

Comment (dgordon-ch) [reply dao/mongo.go:10]: In flight with Dileep.

File: go.mod [hunk 2]
```diff
@@ -9,7 +9,7 @@ require (
 	github.com/companieshouse/go-sdk-manager v0.1.12
 	github.com/companieshouse/go-session-handler v0.1.5
 	github.com/companieshouse/gofigure v0.1.4
-	github.com/companieshouse/lfp-pay-api-core v1.0.7
+	github.com/companieshouse/penalty-payment-api-core v1.0.7
```
Comment (dgordon-ch) [inline go.mod:12]: Does a rename precipitate a new version in pipeline for this? Would it be bumped to 1.0.8?

**Review by dgordon-ch — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (dnair-ch) [reply go.mod:12]: yes I will update this

**Pull request merged**